Client/Frontend/Browser/Tab.swift

```diff
@@ -76,7 +76,7 @@ class Tab: NSObject {
     
     var readerModeAvailable: Bool {
         if let readerMode = self.getHelper(name: "ReaderMode") as? ReaderMode {
-            return readerMode.state == .available
+            return readerMode.state != .unavailable
```

Comment:
call it `readerModeAvailableOrActive` maybe?
Without reading the code, given there is a state `available` for reader mode, I would have expected `readerModeAvailable` to correlate to that only.